exporter/sapmexporter/metadata.yaml

```diff
@@ -11,8 +11,4 @@ status:
 
 tests:
   expect_consumer_error: true
-  goleak:
-    ignore:
-      top:
-        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
-        - "go.opencensus.io/stats/view.(*worker).start"
\ No newline at end of file
+  goleak:
```

Comment:
No need to have it as empty, enabled by default. Also add the new line.

Reply:
thanks, done.